x-pack/plugins/uptime/public/lib/alert_types/monitor_status.tsx

```diff
@@ -54,11 +53,8 @@ export const initMonitorStatusAlertType: AlertTypeInitializer = ({
     reason: fields.reason,
     link: getMonitorRouteFromMonitorId({
       monitorId: fields['monitor.id']!,
-      dateRangeEnd:
-        fields['kibana.rac.alert.status'] === 'open' ? 'now' : fields['kibana.rac.alert.end']!,
-      dateRangeStart: moment(new Date(fields['kibana.rac.alert.start']!))
-        .subtract('5', 'm')
-        .toISOString(),
+      dateRangeEnd: fields[ALERT_STATUS] === 'open' ? 'now' : fields[ALERT_END]!,
```

Comment:
Thank you for doing this. I actually have a branch up getting ready to update this. Much appreciated.